src/librustdoc/html/render/mod.rs

```diff
@@ -1646,16 +1654,27 @@ fn render_rightside(
     cx: &Context<'_>,
     item: &clean::Item,
     containing_item: &clean::Item,
+    render_mode: RenderMode,
 ) {
     let tcx = cx.tcx();
 
+    let const_stable_since;
+    // FIXME: Once https://github.com/rust-lang/rust/issues/67792 is merged, we can remove
+    // this condition.
+    let (const_stability, const_stable_since) = if render_mode == RenderMode::Normal {
+        const_stable_since = containing_item.const_stable_since(tcx);
+        (item.const_stability(tcx), const_stable_since.as_deref())
```

Comment:
Why the need for an `as_deref()` and the `const_stable_since` variable that is immediately shadowed? Can you just use the Symbol below instead of converting here?

Reply:
We need to convert it to `&str` in any in `render_stability_since_raw` because we need to compare it to another `&str`.

Reply:
Symbols are designed to be fast to compare to each other, and the other `&str` seems to be from a Symbol as well. So I think we should change that function, though probably in a different PR.

Anyway, can't you do `.map(|s| s.as_str())` at the call-site of `render_stability_since_raw` to avoid this messy code?